ixdiagnose/artifacts/coredumps.py

```diff
@@ -0,0 +1,62 @@
+import os
```

Comment:
Now that I think about this functionality. This is almost guaranteed to never work. Most core dumps don't compress very well and are guaranteed to be > 10MB. A worker in our process pool is multi-hundred megabytes.....(which is what I see crash a lot of the time).

I think we just need to remove this entire file. The reason why is 2-fold:
1. we generate an alert in the webUI when a core dump occurs and ask the user to upload it to us.
2. in our current debug program, we run `system.coredumps` to get information about the core dumps that have occurred on their system.

Let's not use this module for now. If you want to keep it here but not load it, that's fine.